### PR TITLE
added dashboard template and static files, added bootstrap4

### PR DESCRIPTION
**Pull request by danielabertranou-eb**

### CODE REVIEW

File: login_app/templates/login_app/login.html [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 <form method="post">
    <div class="container login-container">
-     <div class="row text-center">
-       <div class="row"style="margin: 10px"> Do you have another account to login?</div>
-       <a class="btn" style="color: #ffffff; background-color: #ffa500" href="{% url "social:begin" "eventbrite" %}">Eventbrite</a>
+
+       <div class="row text-center" style="margin: 10px"> Do you want to login with your Eventbrite account?</div>
+       <br>
+       <div class="row" style="margin: 10px">
+         <a class="btn" style="color: #FFF; background-color: #D1410C" href="{% url 'social:begin' 'eventbrite' %}">Eventbrite</a>
```
Comment (eb-fmezzabotta) [inline login_app/templates/login_app/login.html]: Inline styling is usually not a good idea. Can we make `class`es out of these?

File: login_app/templates/login_app/login.html [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
+       </div>
 	  </div>
```
Comment (eb-fmezzabotta) [inline login_app/templates/login_app/login.html]: Please fix the indentation of this line.

File: revenue_app/templates/revenue_app/_footer.html [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
```
Comment (eb-fmezzabotta) [inline revenue_app/templates/revenue_app/_footer.html]: nit: emtpy line!

File: revenue_app/urls.py [hunk 4]
```diff
@@ -1,7 +1,7 @@
 from django.conf.urls import url
-from .views import Index
+from .views import Dashboard
```
Comment (eb-fmezzabotta) [inline revenue_app/urls.py]: Unless the path is _too_ long, it's usually a good idea to use absolute paths as opposed to relative paths.

File: login_app/templates/login_app/login.html (hunk 1 above)
Comment (facundochaud-eb) [reply login_app/templates/login_app/login.html]: there's a `.m-2` or `.m-3` class that would set `margin: 8px` or `margin: 12px`

Comment (danielabertranou-eb) [reply login_app/templates/login_app/login.html]: see #8

File: login_app/templates/login_app/login.html (hunk 2 above)
Comment (danielabertranou-eb) [reply login_app/templates/login_app/login.html]: see #8

File: revenue_app/templates/revenue_app/_footer.html (hunk 3 above)
Comment (danielabertranou-eb) [reply revenue_app/templates/revenue_app/_footer.html]: see #8

File: revenue_app/urls.py (hunk 4 above)
Comment (danielabertranou-eb) [reply revenue_app/urls.py]: see #8

**Pull request merged**